src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -585,7 +606,7 @@ def load_status_file_components(self, initial_load=False):
         self.__high_level_status_message = status_file_data['status']['formattedMessage']['message']
         for i in range(0, len(status_file_data['status']['substatus'])):
             name = status_file_data['status']['substatus'][i]['name']
-            if name == Constants.PATCH_INSTALLATION_SUMMARY:     # if it exists, it must be to spec, or an exception will get thrown
```

Comment:
Avoid changes that cause PR overhead on big PRs.

If there's a reason to change these things, a separate PR post discussion is better where everything light-weight is scanned in one easy pass and completed.

These changes will need to get visually scanned by the PR reviewer in every iteration as it stands.